pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -472,6 +473,13 @@ func (r *ReconcileClusterPool) addClusters(
 		errs = append(errs, fmt.Errorf("%s: %w", pullSecretDependent, err))
 	}
 
+	// Load install config templates if specified
+	installConfigTemplate, err := r.getInstallConfigTemplate(clp, logger)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("%s: %w", icSecretDependent, err))
+	}
+
+	// Not sure if we still need a cloudbuilder if we're getting a fully spec'ed install-config.yaml
```

Comment:
I think we do, for various other objects we generate in those builders, we're really just skipping one. Safe to drop this comment.